### DISCUSSION TITLE
which site to choose language programming c/c++ for Termux

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#11737] [Q&A] [answered]
Recommended website for programming c/c++ Termux app

**Comment by ChiefMikeK** [comment, marked as answer]
You should more fully explain what you are trying to acomplish and try search/asking over on Reddit /r/cpp , /r/c /r/??? then you could crosspost to /r/termux 👍 
> https://www.reddit.com/search/?q=c%2B%2B
